<!DOCTYPE html>
<html lang="en"><head>
<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/tabby.min.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/light-border.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.3.353">

  <meta name="author" content="Transmission zero">
  <title>Homing gene drive journal club</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
      vertical-align: middle;
    }
  </style>
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/theme/quarto.css">
  <link rel="stylesheet" href="styles.css">
  <link href="index_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-titled.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-titled) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-titled.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-titled .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-titled .callout-title  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-titled.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-titled.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-title {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-title {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-title {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-titled .callout-body > .callout-content > :last-child {
    margin-bottom: 0.5rem;
  }

  .callout.callout-titled .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-titled) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-title {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-title {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-title {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-title {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-title {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
  <script src="index_files/libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
  <script src="index_files/libs/plotly-binding-4.10.2/plotly.js"></script>
  <script src="index_files/libs/typedarray-0.1/typedarray.min.js"></script>
  <script src="index_files/libs/jquery-3.5.1/jquery.min.js"></script>
  <link href="index_files/libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
  <script src="index_files/libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
  <link href="index_files/libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet">
  <script src="index_files/libs/plotly-main-2.11.1/plotly-latest.min.js"></script>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Homing gene drive journal club</h1>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
Transmission zero 
</div>
</div>
</div>

</section>
<section id="what-is-this-presentation" class="slide level2">
<h2>What is this presentation?</h2>
<ul>
<li>Slides are generated in R from a meta-analysis data set and screenshots from the paper</li>
<li>Feedback on my interpretation of the paper and how best to present it is very much appreciated</li>
<li>In addition to these JCs, we intended to send slides to the original authors to allow them the opportunity to find mistakes in our interpretation of their data</li>
<li>Suggestions you make can be simultaneously applied to the code for &gt;80 papers: https://docs.google.com/spreadsheets/d/1kzc8SkY2fb0NEeGcFSwGPR9j_cCKWgiv9lECuCD97UI/edit?usp=sharing</li>
<li>The same analysis pipeline and figures are used for a manuscript in preparation and a webtool: https://sverkuijl.shinyapps.io/GeneDrive/</li>
<li>Currently the meta-analysis team is Sebald Verkuijl, Alessa Weiler, Edward Ivimey-Cook, Nikolai Windbichler, Philip Leftwich, Michael Bonsall</li>
</ul>
</section>
<section id="paper-title-and-authors." class="slide level2">
<h2>Paper title and authors.</h2>

<img data-src="Harvey-Samuel2023/T01%20Paper%20title%20and%20authors..PNG" class="r-stretch quarto-figure-center"></section>
<section id="female-southern-house-mosquito-culex-quinquefasciatus-say-ovipositing-an-egg-raft." class="slide level2">
<h2>Female southern house mosquito, Culex quinquefasciatus Say, ovipositing an egg raft.</h2>

<img data-src="Harvey-Samuel2023/T10%20Female%20southern%20house%20mosquito,%20Culex%20quinquefasciatus%20Say,%20ovipositing%20an%20egg%20raft..png" class="r-stretch quarto-figure-center"></section>
<section id="culex-quinquefasciatus-is-the-main-vector-of-the-human-filarial-nematode-wuchereria-bancrofti-throughout-the-tropical-and-subtropical-world" class="slide level2">
<h2>Culex quinquefasciatus is the main vector of the human filarial nematode, Wuchereria bancrofti, throughout the tropical and subtropical world</h2>

<img data-src="Harvey-Samuel2023/T15%20Culex%20quinquefasciatus%20is%20the%20main%20vector%20of%20the%20human%20filarial%20nematode,%20Wuchereria%20bancrofti,%20throughout%20the%20tropical%20and%20subtropical%20world.png" class="r-stretch quarto-figure-center"></section>
<section id="cx.-quinquefasciatus-can-transmit-major-avian-pathogens-avian-malaria-west-nile-virus-avian-poxviruses" class="slide level2">
<h2>Cx. quinquefasciatus can transmit major avian pathogens (Avian malaria, West Nile virus, Avian poxviruses)</h2>

<img data-src="Harvey-Samuel2023/T20%20Cx.%20quinquefasciatus%20can%20transmit%20major%20avian%20pathogens%20(Avian%20malaria,%20West%20Nile%20virus,%20Avian%20poxviruses).png" class="r-stretch quarto-figure-center"></section>
<section id="gene-drives-can-spread-autonomously-from-low-to-high-frequency-in-a-population-despite-not-conferring-a-fitness-advantage" class="slide level2">
<h2>Gene drives can spread autonomously from low to high frequency in a population, despite not conferring a fitness advantage</h2>

<img data-src="Harvey-Samuel2023/T25%20Gene%20drives%20can%20spread%20autonomously%20from%20low%20to%20high%20frequency%20in%20a%20population,%20despite%20not%20conferring%20a%20fitness%20advantage.png" class="r-stretch quarto-figure-center"></section>
<section id="homing-gene-drive-inheritance-rates-for-similar-transgenes-have-varied-between-species." class="slide level2">
<h2>Homing gene drive inheritance rates for similar transgenes have varied between species.</h2>

<img data-src="Harvey-Samuel2023/T50%20Homing%20gene%20drive%20inheritance%20rates%20for%20similar%20transgenes%20have%20varied%20between%20species..PNG" class="r-stretch quarto-figure-center"></section>
<section id="schematic-outline-of-targeted-gene-locations-in-cx.-quinquefasciatus-chromosome-i" class="slide level2">
<h2>Schematic outline of targeted gene locations in Cx. quinquefasciatus chromosome I</h2>

<img data-src="Harvey-Samuel2023/G01%20Schematic%20outline%20of%20targeted%20gene%20locations%20in%20Cx.%20quinquefasciatus%20chromosome%20I.png" class="r-stretch quarto-figure-center"></section>
<section id="photo-of-a-transgenic-lines" class="slide level2">
<h2>Photo of a transgenic lines</h2>

<img data-src="Harvey-Samuel2023/G05%20Photo%20of%20a%20transgenic%20lines.png" class="r-stretch quarto-figure-center"></section>
<section id="presence-of-the-backbone-in-the-transgene-suggesting-the-tandem-insertions." class="slide level2">
<h2>Presence of the backbone in the transgene, suggesting the tandem insertions.</h2>

<img data-src="Harvey-Samuel2023/G07%20Presence%20of%20the%20backbone%20in%20the%20transgene,%20suggesting%20the%20tandem%20insertions..PNG" class="r-stretch quarto-figure-center"></section>
<section id="high-level-nuclease-and-target-features-for-all-crosses" class="slide level2">
<h2>High level nuclease and target features for all crosses</h2>

<img data-src="index_files/figure-revealjs/unnamed-chunk-6-1.png" width="1536" class="r-stretch"></section>
<section id="drive-element-dna-maps-blue-url-and-attributes" class="slide level2 smaller">
<h2>Drive element DNA maps (blue = url) and attributes</h2>
<table>
<colgroup>
<col style="width: 1%">
<col style="width: 5%">
<col style="width: 17%">
<col style="width: 17%">
<col style="width: 28%">
<col style="width: 3%">
<col style="width: 4%">
<col style="width: 5%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 4%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">&nbsp;</th>
<th style="text-align: center;">Papers this element appears in</th>
<th style="text-align: center;">Target locus</th>
<th style="text-align: center;">Drive</th>
<th style="text-align: center;">Drive:Target aligned (these Benchling links have issues)</th>
<th style="text-align: center;">Drive allele size</th>
<th style="text-align: center;">gRNA promoter(s)</th>
<th style="text-align: center;">gRNA target seq</th>
<th style="text-align: center;">gRNA scaffold</th>
<th style="text-align: center;">non aligning 5’ seq</th>
<th style="text-align: center;">non aligning 3’ seq</th>
<th style="text-align: center;">gRNA PAM orientation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><strong>21</strong></td>
<td style="text-align: center;">Harvey-Samuel2023</td>
<td style="text-align: center;"><a href="https://benchling.com/s/seq-GeVY2wMfgGlT3leZGHk3?m=slm-XyopIZd0ChVWU7tHWbqB">white</a></td>
<td style="text-align: center;"><a href="https://benchling.com/s/seq-3BLzcTAu2YUKxarjjq42?m=slm-ICQxhGu6USVG57n4C1vH">Drive</a></td>
<td style="text-align: center;"><a href="https://benchling.com/san/f/lib_04NM5QTr-harvey-samuel2023/seq_glK1YxbD-pvmg0295_cquin_w-inserted/edit?alignment=seqanl_Qhav4nLO">Drive:Target</a></td>
<td style="text-align: center;">4198</td>
<td style="text-align: center;">A1:CqU6.653(CPIJ039653)</td>
<td style="text-align: center;">A1:CGTGATCAACGTGTTCT/CCGCGG</td>
<td style="text-align: center;">A1:WT-76nt_O+5bp</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
<td style="text-align: center;">3’</td>
</tr>
<tr class="even">
<td style="text-align: center;"><strong>118</strong></td>
<td style="text-align: center;">Harvey-Samuel2023</td>
<td style="text-align: center;"><a href="https://benchling.com/s/seq-kLDLQFv2i5vP7AXtI7dC?m=slm-2SBDVb25CovGPNxcZjLj">kmo</a></td>
<td style="text-align: center;"><a href="https://benchling.com/s/seq-Jh5gVDZzX2ECuCUPmsKU?m=slm-04qcNIzWB8if2hARSODf">Drive</a></td>
<td style="text-align: center;"><a href="https://benchling.com/san/f/lib_04NM5QTr-harvey-samuel2023/seq_w6TWtjqy-agg2069-inserted/edit?alignment=seqanl_0nJhobZv">Drive:Target</a></td>
<td style="text-align: center;">3429</td>
<td style="text-align: center;">A1:Cq7SK.933(CPIJ039933)</td>
<td style="text-align: center;">A1:GTACAGTGCGGTCCGCA/AGGAGG</td>
<td style="text-align: center;">A1:BB25-80nt</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
<td style="text-align: center;">3’</td>
</tr>
</tbody>
</table>
</section>
<section id="nuclease-element-dna-maps-blue-url-and-attributes" class="slide level2 smaller">
<h2>Nuclease element DNA maps (blue = url) and attributes</h2>
<table style="width:100%;">
<colgroup>
<col style="width: 3%">
<col style="width: 13%">
<col style="width: 13%">
<col style="width: 43%">
<col style="width: 11%">
<col style="width: 8%">
<col style="width: 7%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">&nbsp;</th>
<th style="text-align: center;">Papers this element appears in</th>
<th style="text-align: center;">Nuclease insertion index/locus</th>
<th style="text-align: center;">Nuclease carrying element</th>
<th style="text-align: center;">Promoter + 5UTR size (bp)</th>
<th style="text-align: center;">Nuclease codon opt</th>
<th style="text-align: center;">3 UTR size (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><strong>21</strong></td>
<td style="text-align: center;">Harvey-Samuel2023</td>
<td style="text-align: center;">cardinal.Feng2021</td>
<td style="text-align: center;"><a href="https://benchling.com/s/seq-1Slypxb3W3TrdBvthm7B?m=slm-pmtb7qLGNHph8edl9M0h">vasa_Cas9-NLS_vasa</a></td>
<td style="text-align: center;">2415</td>
<td style="text-align: center;">Unk</td>
<td style="text-align: center;">762</td>
</tr>
</tbody>
</table>
</section>
<section id="cross-types-sexnucleasegrna" class="slide level2">
<h2>Cross types (sex/nuclease;gRNA)</h2>

<img data-src="index_files/figure-revealjs/unnamed-chunk-9-1.png" width="1536" class="r-stretch"></section>
<section id="genetic-cross-scheme-for-the-m-to-m-homing-analysis" class="slide level2">
<h2>Genetic cross scheme for the M-to-m homing analysis</h2>

<img data-src="Harvey-Samuel2023/I01%20Genetic%20cross%20scheme%20for%20the%20M-to-m%20homing%20analysis.png" class="r-stretch quarto-figure-center"></section>
<section id="genetic-cross-scheme-of-the-m-to-m-homing-analysis" class="slide level2">
<h2>Genetic cross scheme of the m-to-m homing analysis</h2>

<img data-src="Harvey-Samuel2023/I03%20Genetic%20cross%20scheme%20of%20the%20m-to-m%20homing%20analysis.png" class="r-stretch quarto-figure-center"></section>
<section id="inheritance-rates-of-white-grna6-egfp-and-vasa-cas9-dsred-transgenes-in-the-g2-progeny" class="slide level2">
<h2>Inheritance rates of white-gRNA6 (eGFP) and vasa-cas9 (DsRed) transgenes in the G2 progeny</h2>

<img data-src="Harvey-Samuel2023/I10%20Inheritance%20rates%20of%20white-gRNA6%20(eGFP)%20and%20vasa-cas9%20(DsRed)%20transgenes%20in%20the%20G2%20progeny.png" class="r-stretch quarto-figure-center"></section>
<section id="graphical-representation-of-the-experimental-design-which-allows-to-track-donor-and-receiver-chromosomes" class="slide level2">
<h2>Graphical representation of the experimental design, which allows to track donor and receiver chromosomes</h2>

<img data-src="Harvey-Samuel2023/I25%20Graphical%20representation%20of%20the%20experimental%20design,%20which%20allows%20to%20track%20donor%20and%20receiver%20chromosomes.png" class="r-stretch quarto-figure-center"></section>
<section id="genetic-cross-scheme-used-to-test-homing-efficiencies-using-a-marked-chromosome-approach." class="slide level2">
<h2>Genetic cross scheme used to test homing efficiencies using a marked chromosome approach.</h2>

<img data-src="Harvey-Samuel2023/I30%20Genetic%20cross%20scheme%20used%20to%20test%20homing%20efficiencies%20using%20a%20marked%20chromosome%20approach..png" class="r-stretch quarto-figure-center"></section>
<section id="overall-inheritance-rates-c-and-homing-rates-d-calculated-using-the-receiver-chromosomes-only-in-the-marked-chromosome-strategy" class="slide level2">
<h2>Overall inheritance rates (c) and homing rates (d) calculated using the receiver chromosomes only, in the marked-chromosome strategy</h2>

<img data-src="Harvey-Samuel2023/I35%20Overall%20inheritance%20rates%20(c)%20and%20homing%20rates%20(d)%20calculated%20using%20the%20receiver%20chromosomes%20only,%20in%20the%20marked-chromosome%20strategy.png" class="r-stretch quarto-figure-center"></section>
<section id="genetic-cross-scheme-for-the-kmo-grna-transgene-and-vasa-cas9" class="slide level2">
<h2>Genetic cross scheme for the kmo-gRNA transgene and vasa-Cas9</h2>

<img data-src="Harvey-Samuel2023/I50%20Genetic%20cross%20scheme%20for%20the%20kmo-gRNA%20transgene%20and%20vasa-Cas9.png" class="r-stretch quarto-figure-center"></section>
<section id="the-inheritance-rates-of-kmo-grna-marked-by-hr5ie1-dsred-and-vasa-cas9-marked-by-opie2-dsred-transgenes-in-the-g2-progeny" class="slide level2">
<h2>The inheritance rates of kmo-gRNA (marked by Hr5IE1-DsRed) and vasa-Cas9 (marked by Opie2-DsRed) transgenes in the G2 progeny</h2>

<img data-src="Harvey-Samuel2023/I60%20The%20inheritance%20rates%20of%20kmo-gRNA%20(marked%20by%20Hr5IE1-DsRed)%20and%20vasa-Cas9%20(marked%20by%20Opie2-DsRed)%20transgenes%20in%20the%20G2%20progeny.png" class="r-stretch quarto-figure-center"></section>
<section id="deposited-rnp-leads-to-kmo-mosaicism-in-g2-progeny" class="slide level2">
<h2>Deposited RNP leads to kmo mosaicism in G2 progeny</h2>

<img data-src="Harvey-Samuel2023/I70%20Deposited%20RNP%20leads%20to%20kmo%20mosaicism%20in%20G2%20progeny.png" class="r-stretch quarto-figure-center"></section>
<section id="what-gene-drive-associated-metrics-were-we-able-to-record-for-this-paper-null-absent" class="slide level2 smaller">
<h2>What gene drive associated metrics were we able to record for this paper (null = absent)</h2>
<table>
<colgroup>
<col style="width: 21%">
<col style="width: 41%">
<col style="width: 37%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Metric</th>
<th style="text-align: center;">Description</th>
<th style="text-align: center;">Values.in.paper</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Drive allele inheritance rate</td>
<td style="text-align: center;">The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</td>
<td style="text-align: center;"><font color="color:green">Pos<br>Neg</font></td>
</tr>
<tr class="even">
<td style="text-align: center;">Drive allele inheritance rate of non-standard crosses</td>
<td style="text-align: center;">The rate of drive allele inheritance under non-standard crossing conditions, such as intercrossing with another drive carrier, pairing the drive allele with a cut-resistant allele, homozygosity for the drive allele, or in the absence of Cas9 expressing elements.</td>
<td style="text-align: center;">Null</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Drive allele inheritance (Shadow drive)</td>
<td style="text-align: center;">The rate of drive allele inheritance that would only be expected to occur through shadow drive. Shadow drive occurs when deposited nuclease components mediate inheritance bias in the germline of the deposition recipient</td>
<td style="text-align: center;">Null</td>
</tr>
<tr class="even">
<td style="text-align: center;">Recipient chromosomes with a drive allele</td>
<td style="text-align: center;">In a hemizygous drive carrier, the drive allele is designed to copy itself from the “donor” chromosome to the “recipient” chromosome. In some studies, the homologous chromosomes can be differentiated by a marker separate from the drive allele. This metric reports the fraction of recipient chromosomes carrying the drive allele out of all recipient chromosomes.</td>
<td style="text-align: center;"><font color="color:green">Null<br>Pos<br>Neg</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Drive to r2 alleles on recipient chromosome</td>
<td style="text-align: center;">Drive to r2 alleles on recipient chromosome</td>
<td style="text-align: center;">Null</td>
</tr>
<tr class="even">
<td style="text-align: center;">Donor chromsome inheritance rate</td>
<td style="text-align: center;">Donor chromsome inheritance rate</td>
<td style="text-align: center;"><font color="color:green">Null<br>Pos<br>Neg</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Somatic phenotype rate</td>
<td style="text-align: center;">Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</td>
<td style="text-align: center;"><font color="color:green">Null<br>Neg<br>Pos</font></td>
</tr>
<tr class="even">
<td style="text-align: center;">non-wildtype alleles</td>
<td style="text-align: center;">non-wildtype alleles. This includes functional and non-functy</td>
<td style="text-align: center;">Null</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Target gene disruption rate</td>
<td style="text-align: center;">Target gene disruption rate</td>
<td style="text-align: center;">Null</td>
</tr>
<tr class="even">
<td style="text-align: center;">Resistance alleles classified as functional</td>
<td style="text-align: center;">Resistance alleles classified as functional</td>
<td style="text-align: center;">Null</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Female F2s</td>
<td style="text-align: center;">The fraction of female F2 progeny.</td>
<td style="text-align: center;">Null</td>
</tr>
<tr class="even">
<td style="text-align: center;">Nuclease transgene inheritance</td>
<td style="text-align: center;">Nuclease transgene inheritance</td>
<td style="text-align: center;"><font color="color:green">Null<br>Pos<br>Neg</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Drive:Marker recombination rate</td>
<td style="text-align: center;">Drive:Marker recombination rate</td>
<td style="text-align: center;">Null</td>
</tr>
</tbody>
</table>
</section>
<section id="the-subset-of-factors-we-deem-essential-for-comparing-data-next-slides-are-figures-of-those-that-vary" class="slide level2 smaller">
<h2>The subset of factors we deem essential for comparing data (next slides are figures of those that vary)</h2>
<table>
<colgroup>
<col style="width: 17%">
<col style="width: 44%">
<col style="width: 37%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Factor</th>
<th style="text-align: center;">Description</th>
<th style="text-align: center;">Levels</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Publication</td>
<td style="text-align: center;">A unique short identifier of the source publication.</td>
<td style="text-align: center;">Harvey-Samuel2023</td>
</tr>
<tr class="even">
<td style="text-align: center;">Misc.Factor</td>
<td style="text-align: center;">A grouping of miscellaneous differences between crosses that isn’t fully captured by other factors in the dataset. If something is repeatedly investigated within the dataset, a new dedicated factor will be created.</td>
<td style="text-align: center;">NA</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Species</td>
<td style="text-align: center;">The name of the organism being studied.</td>
<td style="text-align: center;">C. quinquefasciatus</td>
</tr>
<tr class="even">
<td style="text-align: center;">Strain</td>
<td style="text-align: center;">Description of the strain used in cross.</td>
<td style="text-align: center;"><font color="color:green">California<br>Unk</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Nuclease.Insertion.Variant</td>
<td style="text-align: center;">An identifier of equivalent nuclease expressing elements inserted at different loci. Note that we do not consider independently derived HDR events at the same locus as insertion variants. If a transgene is reused in a subsequent study, we use the same insertion identifier from the first publication.</td>
<td style="text-align: center;">cardinal.Feng2021</td>
</tr>
<tr class="even">
<td style="text-align: center;">D1.Nuc.Pheno</td>
<td style="text-align: center;">A phenotype associated with nuclease activity. Most often, a phenotype that is informative about the cutting of the target allele in an individual heterozygous for the drive allele and a target allele.</td>
<td style="text-align: center;"><font color="color:green">No<br>Yes_Mos</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">D1.Sex</td>
<td style="text-align: center;">D1 is the F1 parent that is being evaluated, and D1.Sex is the sex of that group/individual. In almost all cases, one sex is of interest for the gene drive process in the cross (e.g., drive-carrying males crossed to wild-type females). In most crosses, this is an individual or group of individuals that are double heterozygous for the gRNA and Cas9 drive element. For preparation / control crosses (e.g., gRNA individuals crossed to Cas9 carriers), the sex carrying the element designed to be biased is designated as D1.</td>
<td style="text-align: center;"><font color="color:green">Male<br>Female</font></td>
</tr>
<tr class="even">
<td style="text-align: center;">T1.Cutting.Expected</td>
<td style="text-align: center;">The expectation of whether a target allele in individual could be cut. This only takes into account the presence of Cas9 and at least one gRNA gene, and the presence of a valid target.</td>
<td style="text-align: center;">No drive allele</td>
</tr>
<tr class="odd">
<td style="text-align: center;">D0.Nuclease.Sex</td>
<td style="text-align: center;">The sex of the nuclease-carrying grandparent.</td>
<td style="text-align: center;"><font color="color:green">Male<br>Female</font></td>
</tr>
<tr class="even">
<td style="text-align: center;">D0.NG_Unsorted</td>
<td style="text-align: center;">A combination of the D0 nuclease/gRNA descriptions indepedent of sex. This means that if the sexes of the nuclease carrying D0 grandparents are swapped, the value is the same.</td>
<td style="text-align: center;"><font color="color:green">Nuc;gRNA X -;-<br>Nuc;- X -;gRNA</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">F2.Nuc.Pheno.Locus</td>
<td style="text-align: center;">The locus identifier associated with the nuclease activity phenotype.</td>
<td style="text-align: center;">A</td>
</tr>
<tr class="even">
<td style="text-align: center;">DoubleEntryNote</td>
<td style="text-align: center;">A note on progeny that are represented multiple times within the same cross. This is sometimes needed when two metrics have been measured for the same group of progeny but cannot be simultaneously assigned. For example: Overall Cas9 and gRNA inheritance rates are known, but information of what</td>
<td style="text-align: center;">NA</td>
</tr>
<tr class="odd">
<td style="text-align: center;">F2.Nuc.Pheno.Gene</td>
<td style="text-align: center;">The gene associated with the nuclease activity phenotype.</td>
<td style="text-align: center;"><font color="color:green">white<br>kmo</font></td>
</tr>
<tr class="even">
<td style="text-align: center;">F2.Nuc.Pheno.FitType</td>
<td style="text-align: center;">A description of the type of phenotype associated with the nuclease activity phenotype.</td>
<td style="text-align: center;">Somatic Phenotype</td>
</tr>
<tr class="odd">
<td style="text-align: center;">F2.Nuc.Pheno.Sufficiency</td>
<td style="text-align: center;">A note on whether the nuclease activity phenotype gene is expected to be haplosufficent or haploinsufficient.</td>
<td style="text-align: center;">Haplosufficient</td>
</tr>
<tr class="even">
<td style="text-align: center;">Nuclease.Promoter.5UTR</td>
<td style="text-align: center;">The nuclease promoter and the 5’UTR involved in this cross.</td>
<td style="text-align: center;">vasa</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Nuclease.Gene</td>
<td style="text-align: center;">The nuclease gene involved in this cross.</td>
<td style="text-align: center;">Cas9</td>
</tr>
<tr class="even">
<td style="text-align: center;">D.Target.Gene</td>
<td style="text-align: center;">The gene/locus targeted by the gRNAs/DNA-binding protein involved in this cross.</td>
<td style="text-align: center;"><font color="color:green">white<br>kmo</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">D.Target.Fitness</td>
<td style="text-align: center;">A description of the expected consequence of loss of function of the target gene.</td>
<td style="text-align: center;">Putative Neutral</td>
</tr>
<tr class="even">
<td style="text-align: center;">D.Target.Sufficiency</td>
<td style="text-align: center;">A note on whether the target gene is expected to be haplosufficent or haploinsufficient.</td>
<td style="text-align: center;">Haplosufficient</td>
</tr>
<tr class="odd">
<td style="text-align: center;">D.gRNA.Target.Sequence</td>
<td style="text-align: center;">The DNA-binding protein or gRNA+PAM target sequence. The expected cut site is indicated with a “/” symbol.</td>
<td style="text-align: center;"><font color="color:green">A1:CGTGATCAACGTGTTCT/CCGCGG <br>A1:GTACAGTGCGGTCCGCA/AGGAGG </font></td>
</tr>
<tr class="even">
<td style="text-align: center;">D.gRNA.Target.Site</td>
<td style="text-align: center;">A description of the functional significance of gRNA target sequence within the target gene.</td>
<td style="text-align: center;">A1: CDS</td>
</tr>
<tr class="odd">
<td style="text-align: center;">D.gRNA.Promoter</td>
<td style="text-align: center;">A list of the gRNA promoters involved in this cross. The names and sequences of the promoters are matched to those described in Anderson et al.&nbsp;2020.</td>
<td style="text-align: center;"><font color="color:green">A1:CqU6.653(CPIJ039653) <br>A1:Cq7SK.933(CPIJ039933) </font></td>
</tr>
<tr class="even">
<td style="text-align: center;">F2.Sex</td>
<td style="text-align: center;">The sex of the F2 progeny.</td>
<td style="text-align: center;">Male&amp;Female</td>
</tr>
<tr class="odd">
<td style="text-align: center;">F2.KO.Req</td>
<td style="text-align: center;">Number of alleles required to be disrupted in this F2 genotype to disrupt the function of the target gene. This includes germline mutations where the disruption occurred in the parent.</td>
<td style="text-align: center;"><font color="color:green">rescue<br>2<br>0<br>1</font></td>
</tr>
<tr class="even">
<td style="text-align: center;">F2.NG</td>
<td style="text-align: center;">Notation of the presence of the nuclease gene and at least one gRNA capable of cutting a target allele at the drive locus.</td>
<td style="text-align: center;"><font color="color:green">Unk;gRNA<br>Unk;-<br>Nuc;gRNA<br>-;gRNA<br>Nuc;-<br>-;-</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">F1.Possible.Germline.Mutation.Expected</td>
<td style="text-align: center;">Notation of the possibility of a resistance mutation having been inherited by this F2.</td>
<td style="text-align: center;"><font color="color:green">No<br>Yes</font></td>
</tr>
<tr class="even">
<td style="text-align: center;">F1.Nuclease.Sex</td>
<td style="text-align: center;">The sex of the nuclease-carrying parent.</td>
<td style="text-align: center;"><font color="color:green">Male<br>Female</font></td>
</tr>
<tr class="odd">
<td style="text-align: center;">F1.NG_Unsorted</td>
<td style="text-align: center;">A combination of the D1 nuclease/gRNA descriptions indepedent of sex. This means that if the sexes of the nuclease carrying parent are swapped, the value is the same.</td>
<td style="text-align: center;">Nuc;gRNA X -;-</td>
</tr>
<tr class="even">
<td style="text-align: center;">D1.GESP.Expected</td>
<td style="text-align: center;">Notation of the possibility of a nuclease component being biased in the D1 individual only in cases where cutting may be expected.</td>
<td style="text-align: center;">-;gRNA</td>
</tr>
<tr class="odd">
<td style="text-align: center;">T1.GESP.Expected</td>
<td style="text-align: center;">Notation of the possibility of a nuclease component being biased in the T1 individual only in cases where cutting may be expected.</td>
<td style="text-align: center;">No</td>
</tr>
</tbody>
</table>
</section>
<!-- ## How to interpret the next images? -->
<!-- -   Text explaining how the figures work (get from overleaf) -->
<!-- -   Hover your mouse over a data point for more info -->
<!-- -   Crosses considered replicates are merged -->
<!-- -   Gray lines are drawn between 'paired' crosses -->
<!-- -   Circle size is proportional to the number of progeny scored -->
<!-- -   Only show crosses with 'paired' data for the F2 factors -->
<!-- -   Figure out why the strain isn't showing -->
<section id="drive_allele_inheritance-inclusion-criteria" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance inclusion criteria</h2>

<img data-src="index_files/figure-revealjs/unnamed-chunk-14-1.png" width="960" class="r-stretch"><p>X-axis: In the data selection Sankey diagrams, the green labels and columns indicate values considered positive for the metric being measured. The blue labels and columns indicate values that are negative but are still valid and necessary to calculate an overall rate. Red labels and columns indicate values that cause those F2 counts to be excluded, and black labels and columns indicate neutral values that do not affect the metric calculation. If one or more values cause data to be excluded, the streams between columns representing those progeny are coloured red.</p>
<p>Y-axis: F2 count.</p>
</section>
<section id="drive_allele_inheritanceid" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:ID</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-9dcb80e44738f351b3ce" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-9dcb80e44738f351b3ce">{"x":{"data":[{"x":[6.9468943800311536,null,1.0720762838609517,1.977244230452925,null,6.0957543510012329,null,4.1101168210734613,null,4.8863891248474829,null,3.0070263720117509],"y":[0.51257861635220126,null,0.54800724637681164,0.56259659969088094,null,0.61132075471698111,null,0.59395248380129595,null,0.54859752120026095,null,0.55091103965702037],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.9468943800311536,1.0720762838609517,1.977244230452925,6.0957543510012329,4.1101168210734613,4.8863891248474829,3.0070263720117509],"y":[0.51257861635220126,0.54800724637681164,0.56259659969088094,0.61132075471698111,0.59395248380129595,0.54859752120026095,0.55091103965702037],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104)<br>Total (54.8%:605/1104)","Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (56.3%:364/647)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,15.063581509401839,11.528073795455382,7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,7.5999999999999996],"tickmode":"array","ticktext":["02_Harvey-Samuel2023","03_Harvey-Samuel2023","07_Harvey-Samuel2023","05_Harvey-Samuel2023","06_Harvey-Samuel2023","04_Harvey-Samuel2023","01_Harvey-Samuel2023"],"tickvals":[1,2,3,4.0000000000000009,5,6,7],"categoryorder":"array","categoryarray":["02_Harvey-Samuel2023","03_Harvey-Samuel2023","07_Harvey-Samuel2023","05_Harvey-Samuel2023","06_Harvey-Samuel2023","04_Harvey-Samuel2023","01_Harvey-Samuel2023"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Unique cross ID","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b44ac31ce1":{"x":{},"y":{},"size":{},"type":"scatter"},"52b45a3453a":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b44ac31ce1","visdat":{"52b44ac31ce1":["function (y) ","x"],"52b45a3453a":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Cross Number_A unique short identifier of the source publication.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritancestrain" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:Strain</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-c5ab5dc4298a514c87ed" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-c5ab5dc4298a514c87ed">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289],"y":[0.51257861635220126,null,0.55339805825242716,null,0.61132075471698111,null,0.59395248380129595,null,0.54859752120026095,null,0.55091103965702037],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289],"y":[0.51257861635220126,0.55339805825242716,0.61132075471698111,0.59395248380129595,0.54859752120026095,0.55091103965702037],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Unk","California"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Unk","California"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Strain","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b4417c12db":{"x":{},"y":{},"size":{},"type":"scatter"},"52b43f6b4666":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b4417c12db","visdat":{"52b4417c12db":["function (y) ","x"],"52b43f6b4666":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Description of the strain used in cross.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritanced1.nuc.pheno" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:D1.Nuc.Pheno</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-024bce6914a3a0525205" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-024bce6914a3a0525205">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289],"y":[0.51257861635220126,null,0.55339805825242716,null,0.61132075471698111,null,0.59395248380129595,null,0.54859752120026095,null,0.55091103965702037],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289],"y":[0.51257861635220126,0.55339805825242716,0.61132075471698111,0.59395248380129595,0.54859752120026095,0.55091103965702037],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Yes_Mos","No"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Yes_Mos","No"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Nuclease phenotype of drive parent","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b44b9d4db0":{"x":{},"y":{},"size":{},"type":"scatter"},"52b4125e15d1":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b44b9d4db0","visdat":{"52b44b9d4db0":["function (y) ","x"],"52b4125e15d1":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: A phenotype associated with nuclease activity. Most often, a phenotype that is informative about the cutting of the target allele in an individual heterozygous for the drive allele and a target allele.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritanced1.sex" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:D1.Sex</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-1067ab30c63cbfcf6e96" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-1067ab30c63cbfcf6e96">{"x":{"data":[{"x":[1.9468943800311536,null,1.0720762838609517,null,0.97724423045292497,2.0957543510012329,null,1.1101168210734613,1.8863891248474829],"y":[0.51257861635220126,null,0.55339805825242716,null,0.54859752120026095,0.61132075471698111,null,0.55091103965702037,0.59395248380129595],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,1.0720762838609517,0.97724423045292497,2.0957543510012329,1.1101168210734613,1.8863891248474829],"y":[0.51257861635220126,0.55339805825242716,0.54859752120026095,0.61132075471698111,0.55091103965702037,0.59395248380129595],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,17.752922883359997,7.3695752943754629,19.587540214402548,9.7490316265098027],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Female","Male"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Female","Male"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Sex of the drive carrying parent","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b43f8631b7":{"x":{},"y":{},"size":{},"type":"scatter"},"52b463297876":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b43f8631b7","visdat":{"52b43f8631b7":["function (y) ","x"],"52b463297876":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: D1 is the F1 parent that is being evaluated, and D1.Sex is the sex of that group/individual. In almost all cases, one sex is of interest for the gene drive process in the cross (e.g., drive-carrying males crossed to wild-type females). In most crosses, this is an individual or group of individuals that are double heterozygous for the gRNA and Cas9 drive element. For preparation / control crosses (e.g., gRNA individuals crossed to Cas9 carriers), the sex carrying the element designed to be biased is designated as D1.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritanced0.nuclease.sex" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:D0.Nuclease.Sex</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-9713283386ec8c75cf82" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-9713283386ec8c75cf82">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,2.0957543510012329,null,1.1101168210734613,1.8863891248474829],"y":[0.51257861635220126,null,0.55339805825242716,null,0.59395248380129595,0.61132075471698111,null,0.55091103965702037,0.54859752120026095],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,2.0957543510012329,1.1101168210734613,1.8863891248474829],"y":[0.51257861635220126,0.55339805825242716,0.59395248380129595,0.61132075471698111,0.55091103965702037,0.54859752120026095],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,9.7490316265098027,7.3695752943754629,19.587540214402548,17.752922883359997],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Female","Male"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Female","Male"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Sex of grandparent carrying the nuclease gene","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b41e5f1b6":{"x":{},"y":{},"size":{},"type":"scatter"},"52b41ead7001":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b41e5f1b6","visdat":{"52b41e5f1b6":["function (y) ","x"],"52b41ead7001":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The sex of the nuclease-carrying grandparent.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritanced0.ng_unsorted" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:D0.NG_Unsorted</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-630468c37da32eb36410" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-630468c37da32eb36410">{"x":{"data":[{"x":[1.9468943800311536,null,1.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289],"y":[0.51257861635220126,null,0.55339805825242716,null,0.61132075471698111,null,0.59395248380129595,null,0.54859752120026095,null,0.55091103965702037],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,1.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289],"y":[0.51257861635220126,0.55339805825242716,0.61132075471698111,0.59395248380129595,0.54859752120026095,0.55091103965702037],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Nuc;- X -;gRNA","Nuc;gRNA X -;-"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Nuc;- X -;gRNA","Nuc;gRNA X -;-"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Drive grandparent Nuclease;gRNA cross-type","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b42e262946":{"x":{},"y":{},"size":{},"type":"scatter"},"52b4116f1107":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b42e262946","visdat":{"52b42e262946":["function (y) ","x"],"52b4116f1107":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: A combination of the D0 nuclease/gRNA descriptions indepedent of sex. This means that if the sexes of the nuclease carrying D0 grandparents are swapped, the value is the same.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritancef2.nuc.pheno.gene" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:F2.Nuc.Pheno.Gene</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-be420dfb23a96fa46244" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-be420dfb23a96fa46244">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289],"y":[0.51257861635220126,null,0.55339805825242716,null,0.61132075471698111,null,0.59395248380129595,null,0.54859752120026095,null,0.55091103965702037],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289],"y":[0.51257861635220126,0.55339805825242716,0.61132075471698111,0.59395248380129595,0.54859752120026095,0.55091103965702037],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["kmo","white"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["kmo","white"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"F2 nuclease phenotype gene locus","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b451706209":{"x":{},"y":{},"size":{},"type":"scatter"},"52b4725f732b":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b451706209","visdat":{"52b451706209":["function (y) ","x"],"52b4725f732b":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The gene associated with the nuclease activity phenotype.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritanced.target.gene" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:D.Target.Gene</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-ef62dbfb9393670230c5" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-ef62dbfb9393670230c5">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289],"y":[0.51257861635220126,null,0.55339805825242716,null,0.61132075471698111,null,0.59395248380129595,null,0.54859752120026095,null,0.55091103965702037],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289],"y":[0.51257861635220126,0.55339805825242716,0.61132075471698111,0.59395248380129595,0.54859752120026095,0.55091103965702037],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["kmo","white"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["kmo","white"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Drive target gene","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b461a8574":{"x":{},"y":{},"size":{},"type":"scatter"},"52b44c0a5a09":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b461a8574","visdat":{"52b461a8574":["function (y) ","x"],"52b44c0a5a09":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The gene/locus targeted by the gRNAs/DNA-binding protein involved in this cross.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritanced.grna.target.sequence" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:D.gRNA.Target.Sequence</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-5f5c50a6d1c73e62cb57" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-5f5c50a6d1c73e62cb57">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289],"y":[0.51257861635220126,null,0.55339805825242716,null,0.61132075471698111,null,0.59395248380129595,null,0.54859752120026095,null,0.55091103965702037],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289],"y":[0.51257861635220126,0.55339805825242716,0.61132075471698111,0.59395248380129595,0.54859752120026095,0.55091103965702037],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["A1:GTACAGTGCGGTCCGCA/AGGAGG","A1:CGTGATCAACGTGTTCT/CCGCGG"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["A1:GTACAGTGCGGTCCGCA/AGGAGG","A1:CGTGATCAACGTGTTCT/CCGCGG"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"gRNA target sequence","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b463ce2304":{"x":{},"y":{},"size":{},"type":"scatter"},"52b41c722e29":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b463ce2304","visdat":{"52b463ce2304":["function (y) ","x"],"52b41c722e29":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The DNA-binding protein or gRNA+PAM target sequence. The expected cut site is indicated with a “/” symbol.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="drive_allele_inheritanced.grna.promoter" class="slide level2 smaller">
<h2>Drive_Allele_Inheritance:D.gRNA.Promoter</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-f5a7ab311aa70f4d0ec6" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-f5a7ab311aa70f4d0ec6">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289],"y":[0.51257861635220126,null,0.55339805825242716,null,0.61132075471698111,null,0.59395248380129595,null,0.54859752120026095,null,0.55091103965702037],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289],"y":[0.51257861635220126,0.55339805825242716,0.61132075471698111,0.59395248380129595,0.54859752120026095,0.55091103965702037],"text":["Harvey-Samuel2023_01_D1.M.white (51.3%:163/318)<br>Total (51.3%:163/318)","Harvey-Samuel2023_02_D1.F.white (54.8%:605/1104),<br>Harvey-Samuel2023_03_D1.F.white.marked (56.3%:364/647)<br>Total (55.3%:969/1751)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (61.1%:162/265)<br>Total (61.1%:162/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (59.4%:275/463)<br>Total (59.4%:275/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (54.9%:841/1533)<br>Total (54.9%:841/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (55.1%:1028/1866)<br>Total (55.1%:1028/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[8.0755131021429047,18.974026711951147,7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["A1:Cq7SK.933(CPIJ039933)","A1:CqU6.653(CPIJ039653)"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["A1:Cq7SK.933(CPIJ039933)","A1:CqU6.653(CPIJ039653)"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"gRNA promoter","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Drive allele inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b4250d530d":{"x":{},"y":{},"size":{},"type":"scatter"},"52b463243f4a":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b4250d530d","visdat":{"52b4250d530d":["function (y) ","x"],"52b463243f4a":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: A list of the gRNA promoters involved in this cross. The names and sequences of the promoters are matched to those described in Anderson et al.&nbsp;2020.</p>
<p>Y-axis: The rate at which a specific genetic allele is passed from the D1 parent, which also carries a target allele susceptible to cleavage by nuclease components expressed in that individual, as measured in the F2 offspring.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="homed_recipient_chromosomes-inclusion-criteria" class="slide level2 smaller">
<h2>Homed_Recipient_Chromosomes inclusion criteria</h2>

<img data-src="index_files/figure-revealjs/unnamed-chunk-25-1.png" width="960" class="r-stretch"><p>X-axis: In the data selection Sankey diagrams, the green labels and columns indicate values considered positive for the metric being measured. The blue labels and columns indicate values that are negative but are still valid and necessary to calculate an overall rate. Red labels and columns indicate values that cause those F2 counts to be excluded, and black labels and columns indicate neutral values that do not affect the metric calculation. If one or more values cause data to be excluded, the streams between columns representing those progeny are coloured red.</p>
<p>Y-axis: F2 count.</p>
</section>
<section id="homed_recipient_chromosomesid" class="slide level2 smaller">
<h2>Homed_Recipient_Chromosomes:ID</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-68e8db1ff800f569bcda" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-68e8db1ff800f569bcda">{"x":{"data":[{"x":[0.94689438003115356],"y":[0.040677966101694912],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.94689438003115356],"y":[0.040677966101694912],"text":"Harvey-Samuel2023_03_D1.F.white.marked (4.1%:12/295)<br>Total (4.1%:12/295)","customdata":["https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":7.7770369250128502,"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,1.6000000000000001],"tickmode":"array","ticktext":["03_Harvey-Samuel2023"],"tickvals":[1],"categoryorder":"array","categoryarray":["03_Harvey-Samuel2023"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Unique cross ID","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Recipient chromosomes with a drive allele","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b4739c488d":{"x":{},"y":{},"size":{},"type":"scatter"},"52b4364325b2":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b4739c488d","visdat":{"52b4739c488d":["function (y) ","x"],"52b4364325b2":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Cross Number_A unique short identifier of the source publication.</p>
<p>Y-axis: In a hemizygous drive carrier, the drive allele is designed to copy itself from the “donor” chromosome to the “recipient” chromosome. In some studies, the homologous chromosomes can be differentiated by a marker separate from the drive allele. This metric reports the fraction of recipient chromosomes carrying the drive allele out of all recipient chromosomes.</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="donor_to_recipient_chromsomes-inclusion-criteria" class="slide level2 smaller">
<h2>Donor_to_Recipient_Chromsomes inclusion criteria</h2>

<img data-src="index_files/figure-revealjs/unnamed-chunk-27-1.png" width="960" class="r-stretch"><p>X-axis: In the data selection Sankey diagrams, the green labels and columns indicate values considered positive for the metric being measured. The blue labels and columns indicate values that are negative but are still valid and necessary to calculate an overall rate. Red labels and columns indicate values that cause those F2 counts to be excluded, and black labels and columns indicate neutral values that do not affect the metric calculation. If one or more values cause data to be excluded, the streams between columns representing those progeny are coloured red.</p>
<p>Y-axis: F2 count.</p>
</section>
<section id="donor_to_recipient_chromsomesid" class="slide level2 smaller">
<h2>Donor_to_Recipient_Chromsomes:ID</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-129f838f5a79e7db35d3" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-129f838f5a79e7db35d3">{"x":{"data":[{"x":[0.94689438003115356],"y":[0.54404945904173108],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.94689438003115356],"y":[0.54404945904173108],"text":"Harvey-Samuel2023_03_D1.F.white.marked (54.4%:352/647)<br>Total (54.4%:352/647)","customdata":["https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":11.528073795455382,"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,1.6000000000000001],"tickmode":"array","ticktext":["03_Harvey-Samuel2023"],"tickvals":[1],"categoryorder":"array","categoryarray":["03_Harvey-Samuel2023"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Unique cross ID","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Donor chromsome inheritance rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b411c068a2":{"x":{},"y":{},"size":{},"type":"scatter"},"52b42a9c7e7f":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b411c068a2","visdat":{"52b411c068a2":["function (y) ","x"],"52b42a9c7e7f":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Cross Number_A unique short identifier of the source publication.</p>
<p>Y-axis: Donor chromsome inheritance rate</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotype-inclusion-criteria" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype inclusion criteria</h2>

<img data-src="index_files/figure-revealjs/unnamed-chunk-29-1.png" width="960" class="r-stretch"><p>X-axis: In the data selection Sankey diagrams, the green labels and columns indicate values considered positive for the metric being measured. The blue labels and columns indicate values that are negative but are still valid and necessary to calculate an overall rate. Red labels and columns indicate values that cause those F2 counts to be excluded, and black labels and columns indicate neutral values that do not affect the metric calculation. If one or more values cause data to be excluded, the streams between columns representing those progeny are coloured red.</p>
<p>Y-axis: F2 count.</p>
</section>
<section id="nuclease_induced_phenotypeid" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:ID</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-69781e6497f2a34040f2" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-69781e6497f2a34040f2">{"x":{"data":[{"x":[5.9468943800311536,null,3.0720762838609517,null,4.977244230452925,null,5.0957543510012329,null,4.1101168210734613,null,3.8863891248474829,null,2.0070263720117509,null,2.0981047610985115,null,1.0128587536164559,null,0.98915368382586166],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.9468943800311536,3.0720762838609517,4.977244230452925,5.0957543510012329,4.1101168210734613,3.8863891248474829,2.0070263720117509,2.0981047610985115,1.0128587536164559,0.98915368382586166],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,6.5999999999999996],"tickmode":"array","ticktext":["07_Harvey-Samuel2023","06_Harvey-Samuel2023","02_Harvey-Samuel2023","05_Harvey-Samuel2023","04_Harvey-Samuel2023","01_Harvey-Samuel2023"],"tickvals":[1,2,3,3.9999999999999996,5,6],"categoryorder":"array","categoryarray":["07_Harvey-Samuel2023","06_Harvey-Samuel2023","02_Harvey-Samuel2023","05_Harvey-Samuel2023","04_Harvey-Samuel2023","01_Harvey-Samuel2023"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Unique cross ID","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b47bf73c98":{"x":{},"y":{},"size":{},"type":"scatter"},"52b430053efc":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b47bf73c98","visdat":{"52b47bf73c98":["function (y) ","x"],"52b430053efc":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Cross Number_A unique short identifier of the source publication.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotypef2.ko.req" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:F2.KO.Req</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-c2ec0171ccb878630c2d" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-c2ec0171ccb878630c2d">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,2.0957543510012329,null,1.1101168210734613,null,1.8863891248474829,null,1.0070263720117509,null,2.0981047610985115,null,1.0128587536164559,null,1.9891536838258617],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,2.0957543510012329,1.1101168210734613,1.8863891248474829,1.0070263720117509,2.0981047610985115,1.0128587536164559,1.9891536838258617],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["1","2"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["1","2"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Alleles required to be disrupted for a nuclease phenotype in the absence of cutting","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b478887bac":{"x":{},"y":{},"size":{},"type":"scatter"},"52b44a96589":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b478887bac","visdat":{"52b478887bac":["function (y) ","x"],"52b44a96589":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Number of alleles required to be disrupted in this F2 genotype to disrupt the function of the target gene. This includes germline mutations where the disruption occurred in the parent.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotypef2.ng" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:F2.NG</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-a7c1a3fe528ae300b17e" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-a7c1a3fe528ae300b17e">{"x":{"data":[{"x":[2.9468943800311536,null,3.0720762838609517,null,0.97724423045292497,null,2.0957543510012329,null,1.1101168210734613,null,1.8863891248474829,null,1.0070263720117509,null,2.0981047610985115,null,1.0128587536164559,null,1.9891536838258617],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.9468943800311536,3.0720762838609517,0.97724423045292497,2.0957543510012329,1.1101168210734613,1.8863891248474829,1.0070263720117509,2.0981047610985115,1.0128587536164559,1.9891536838258617],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,3.6000000000000001],"tickmode":"array","ticktext":["-;gRNA","-;-","Unk;-"],"tickvals":[1,2,3],"categoryorder":"array","categoryarray":["-;gRNA","-;-","Unk;-"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"F2 Nuclease;gRNA genes","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b4760f3bed":{"x":{},"y":{},"size":{},"type":"scatter"},"52b41a7632a2":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b4760f3bed","visdat":{"52b4760f3bed":["function (y) ","x"],"52b41a7632a2":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Notation of the presence of the nuclease gene and at least one gRNA capable of cutting a target allele at the drive locus.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotypef1.possible.germline.mutation.expected" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:F1.Possible.Germline.Mutation.Expected</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-a576eb54802a7dd75de7" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-a576eb54802a7dd75de7">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,2.0957543510012329,null,1.1101168210734613,null,1.8863891248474829,null,1.0070263720117509,null,2.0981047610985115,null,1.0128587536164559,null,1.9891536838258617],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,2.0957543510012329,1.1101168210734613,1.8863891248474829,1.0070263720117509,2.0981047610985115,1.0128587536164559,1.9891536838258617],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["No","Yes"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["No","Yes"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"F1.Possible.Germline.Mutation.Expected","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b44fbf25ef":{"x":{},"y":{},"size":{},"type":"scatter"},"52b43f5442f7":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b44fbf25ef","visdat":{"52b44fbf25ef":["function (y) ","x"],"52b43f5442f7":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Notation of the possibility of a resistance mutation having been inherited by this F2.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotypef1.nuclease.sex" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:F1.Nuclease.Sex</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-278284bb3de85113b114" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-278284bb3de85113b114">{"x":{"data":[{"x":[1.9468943800311536,null,1.0720762838609517,null,0.97724423045292497,2.0957543510012329,null,1.1101168210734613,1.8863891248474829,null,1.0070263720117509,2.0981047610985115,null,1.0128587536164559,1.9891536838258617],"y":[0,null,0.11222444889779559,null,0,0,null,1,1,null,0,0,null,1,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,1.0720762838609517,0.97724423045292497,2.0957543510012329,1.1101168210734613,1.8863891248474829,1.0070263720117509,2.0981047610985115,1.0128587536164559,1.9891536838258617],"y":[0,0.11222444889779559,0,0,1,1,0,0,1,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,9.3724780001653851,4.1321831323008702,8.375633856262942,3.1749622531650235,10.121739650034002,5.3666654528257123,9.2398419121274529,4.5807922547838453],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Female","Male"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Female","Male"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Sex of parent carrying the nuclease gene","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b471735d27":{"x":{},"y":{},"size":{},"type":"scatter"},"52b421cd6bc6":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b471735d27","visdat":{"52b471735d27":["function (y) ","x"],"52b421cd6bc6":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The sex of the nuclease-carrying parent.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotypestrain" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:Strain</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-80cc597b0e8365e6a5b2" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-80cc597b0e8365e6a5b2">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289,null,1.0070263720117509,null,1.0981047610985115,null,1.0128587536164559,null,0.98915368382586166],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289,1.0070263720117509,1.0981047610985115,1.0128587536164559,0.98915368382586166],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Unk","California"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Unk","California"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Strain","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b45ac23f22":{"x":{},"y":{},"size":{},"type":"scatter"},"52b45c446cc8":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b45ac23f22","visdat":{"52b45ac23f22":["function (y) ","x"],"52b45c446cc8":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Description of the strain used in cross.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotyped1.nuc.pheno" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:D1.Nuc.Pheno</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-cd600b4caac9ae357155" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-cd600b4caac9ae357155">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289,null,1.0070263720117509,null,1.0981047610985115,null,1.0128587536164559,null,0.98915368382586166],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289,1.0070263720117509,1.0981047610985115,1.0128587536164559,0.98915368382586166],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Yes_Mos","No"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Yes_Mos","No"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Nuclease phenotype of drive parent","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b477ad35ff":{"x":{},"y":{},"size":{},"type":"scatter"},"52b42df77777":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b477ad35ff","visdat":{"52b477ad35ff":["function (y) ","x"],"52b42df77777":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: A phenotype associated with nuclease activity. Most often, a phenotype that is informative about the cutting of the target allele in an individual heterozygous for the drive allele and a target allele.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotyped0.nuclease.sex" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:D0.Nuclease.Sex</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-9fabd18b816303395783" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-9fabd18b816303395783">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,2.0957543510012329,null,1.1101168210734613,1.8863891248474829,null,1.0070263720117509,2.0981047610985115,null,1.0128587536164559,1.9891536838258617],"y":[0,null,0.11222444889779559,null,0,0,null,1,1,null,0,0,null,1,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,2.0957543510012329,1.1101168210734613,1.8863891248474829,1.0070263720117509,2.0981047610985115,1.0128587536164559,1.9891536838258617],"y":[0,0.11222444889779559,0,0,1,1,0,0,1,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,5.3666654528257123,4.1321831323008702,4.5807922547838453,3.1749622531650235,10.121739650034002,9.3724780001653851,9.2398419121274529,8.375633856262942],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Female","Male"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Female","Male"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Sex of grandparent carrying the nuclease gene","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b47eaa2ece":{"x":{},"y":{},"size":{},"type":"scatter"},"52b4665b174":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b47eaa2ece","visdat":{"52b47eaa2ece":["function (y) ","x"],"52b4665b174":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The sex of the nuclease-carrying grandparent.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotyped0.ng_unsorted" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:D0.NG_Unsorted</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-2e6c381e366a20844dfb" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-2e6c381e366a20844dfb">{"x":{"data":[{"x":[1.9468943800311536,null,1.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289,null,1.0070263720117509,null,1.0981047610985115,null,1.0128587536164559,null,0.98915368382586166],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,1.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289,1.0070263720117509,1.0981047610985115,1.0128587536164559,0.98915368382586166],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Nuc;- X -;gRNA","Nuc;gRNA X -;-"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Nuc;- X -;gRNA","Nuc;gRNA X -;-"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Drive grandparent Nuclease;gRNA cross-type","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b44f1062c8":{"x":{},"y":{},"size":{},"type":"scatter"},"52b45c046855":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b44f1062c8","visdat":{"52b44f1062c8":["function (y) ","x"],"52b45c046855":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: A combination of the D0 nuclease/gRNA descriptions indepedent of sex. This means that if the sexes of the nuclease carrying D0 grandparents are swapped, the value is the same.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotypef2.nuc.pheno.gene" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:F2.Nuc.Pheno.Gene</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-6cb1971110cb92ab396c" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-6cb1971110cb92ab396c">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289,null,1.0070263720117509,null,1.0981047610985115,null,1.0128587536164559,null,0.98915368382586166],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289,1.0070263720117509,1.0981047610985115,1.0128587536164559,0.98915368382586166],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["kmo","white"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["kmo","white"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"F2 nuclease phenotype gene locus","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b4783a1a75":{"x":{},"y":{},"size":{},"type":"scatter"},"52b419a372b":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b4783a1a75","visdat":{"52b4783a1a75":["function (y) ","x"],"52b419a372b":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The gene associated with the nuclease activity phenotype.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotyped.target.gene" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:D.Target.Gene</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-4807e52921137317a2ca" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-4807e52921137317a2ca">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289,null,1.0070263720117509,null,1.0981047610985115,null,1.0128587536164559,null,0.98915368382586166],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289,1.0070263720117509,1.0981047610985115,1.0128587536164559,0.98915368382586166],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["kmo","white"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["kmo","white"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Drive target gene","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b470cf743b":{"x":{},"y":{},"size":{},"type":"scatter"},"52b444dd5274":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b470cf743b","visdat":{"52b470cf743b":["function (y) ","x"],"52b444dd5274":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The gene/locus targeted by the gRNAs/DNA-binding protein involved in this cross.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotyped.grna.target.sequence" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:D.gRNA.Target.Sequence</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-b1a89f07909b6090060f" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-b1a89f07909b6090060f">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289,null,1.0070263720117509,null,1.0981047610985115,null,1.0128587536164559,null,0.98915368382586166],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289,1.0070263720117509,1.0981047610985115,1.0128587536164559,0.98915368382586166],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["A1:GTACAGTGCGGTCCGCA/AGGAGG","A1:CGTGATCAACGTGTTCT/CCGCGG"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["A1:GTACAGTGCGGTCCGCA/AGGAGG","A1:CGTGATCAACGTGTTCT/CCGCGG"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"gRNA target sequence","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b4671813f5":{"x":{},"y":{},"size":{},"type":"scatter"},"52b42e104d5d":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b4671813f5","visdat":{"52b4671813f5":["function (y) ","x"],"52b42e104d5d":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The DNA-binding protein or gRNA+PAM target sequence. The expected cut site is indicated with a “/” symbol.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_induced_phenotyped.grna.promoter" class="slide level2 smaller">
<h2>Nuclease_Induced_Phenotype:D.gRNA.Promoter</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-bb35de880580f037603e" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-bb35de880580f037603e">{"x":{"data":[{"x":[1.9468943800311536,null,2.0720762838609517,null,0.97724423045292497,null,1.0957543510012329,null,1.1101168210734613,null,0.88638912484748289,null,1.0070263720117509,null,1.0981047610985115,null,1.0128587536164559,null,0.98915368382586166],"y":[0,null,0.11222444889779559,null,0,null,1,null,0,null,1,null,0,null,1,null,0,null,1],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.9468943800311536,2.0720762838609517,0.97724423045292497,1.0957543510012329,1.1101168210734613,0.88638912484748289,1.0070263720117509,1.0981047610985115,1.0128587536164559,0.98915368382586166],"y":[0,0.11222444889779559,0,1,0,1,0,1,0,1],"text":["Harvey-Samuel2023_01_D1.M.white [A{T/T}; B{(N/T)/T}] (0%:0/155)<br>Total (0%:0/155)","Harvey-Samuel2023_02_D1.F.white [A{T/T}; B{(N/T)/T}] (11.2%:56/499)<br>Total (11.2%:56/499)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/84)<br>Total (0%:0/84)","Harvey-Samuel2023_04_D0.M.D1.M.kmo [A{T/T}; B{T/T}] (100%:50/50)<br>Total (100%:50/50)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{G[A1]/T}; B{T/T}] (0%:0/141)<br>Total (0%:0/141)","Harvey-Samuel2023_05_D0.F.D1.M.kmo [A{T/T}; B{T/T}] (100%:103/103)<br>Total (100%:103/103)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/428)<br>Total (0%:0/428)","Harvey-Samuel2023_06_D0.M.D1.F.kmo [A{T/T}; B{T/T}] (100%:342/342)<br>Total (100%:342/342)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{G[A1]/T}; B{T/T}] (0%:0/499)<br>Total (0%:0/499)","Harvey-Samuel2023_07_D0.F.D1.F.kmo [A{T/T}; B{T/T}] (100%:416/416)<br>Total (100%:416/416)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[5.6286061936452398,10.121739650034002,4.1321831323008702,3.1749622531650235,5.3666654528257123,4.5807922547838453,9.3724780001653851,8.375633856262942,10.121739650034002,9.2398419121274529],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["A1:Cq7SK.933(CPIJ039933)","A1:CqU6.653(CPIJ039653)"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["A1:Cq7SK.933(CPIJ039933)","A1:CqU6.653(CPIJ039653)"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"gRNA promoter","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Somatic phenotype rate","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b44650776":{"x":{},"y":{},"size":{},"type":"scatter"},"52b44539437f":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b44650776","visdat":{"52b44650776":["function (y) ","x"],"52b44539437f":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: A list of the gRNA promoters involved in this cross. The names and sequences of the promoters are matched to those described in Anderson et al.&nbsp;2020.</p>
<p>Y-axis: Somatic phenotypes are changes in the phenotype that are generated by the action of the drive nuclease. In most cases, the somatic phenotype is generated by a haplosufficient endogenous gene into which the gene drive is inserted and which the drive also targets the wild-type version. In individuals heterozygous for the knock-out gene drive allele, embryonic or somatic disruption of the recipient chromosome’s target gene will result in a somatic phenotype. Note that the sex of the drive-carrying parent is not considered for the pairing of somatic phenotype data (it is instead replaced by the sex of the nuclease-carrying parent).</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_carrying_progeny_from_doublehet-inclusion-criteria" class="slide level2 smaller">
<h2>Nuclease_Carrying_Progeny_From_DoubleHet inclusion criteria</h2>

<img data-src="index_files/figure-revealjs/unnamed-chunk-43-1.png" width="960" class="r-stretch"><p>X-axis: In the data selection Sankey diagrams, the green labels and columns indicate values considered positive for the metric being measured. The blue labels and columns indicate values that are negative but are still valid and necessary to calculate an overall rate. Red labels and columns indicate values that cause those F2 counts to be excluded, and black labels and columns indicate neutral values that do not affect the metric calculation. If one or more values cause data to be excluded, the streams between columns representing those progeny are coloured red.</p>
<p>Y-axis: F2 count.</p>
</section>
<section id="nuclease_carrying_progeny_from_doublehetid" class="slide level2 smaller">
<h2>Nuclease_Carrying_Progeny_From_DoubleHet:ID</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-00914276cd2668006caf" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-00914276cd2668006caf">{"x":{"data":[{"x":[3.9468943800311536,null,2.0720762838609517,null,2.977244230452925,null,1.0957543510012329],"y":[0.49433962264150944,null,0.47300215982721383,null,0.49771689497716892,null,0.50964630225080387],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.9468943800311536,2.0720762838609517,2.977244230452925,1.0957543510012329],"y":[0.49433962264150944,0.47300215982721383,0.49771689497716892,0.50964630225080387],"text":["Harvey-Samuel2023_04_D0.M.D1.M.kmo (49.4%:131/265)<br>Total (49.4%:131/265)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (47.3%:219/463)<br>Total (47.3%:219/463)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (49.8%:763/1533)<br>Total (49.8%:763/1533)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (51%:951/1866)<br>Total (51%:951/1866)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[7.3695752943754629,9.7490316265098027,17.752922883359997,19.587540214402548],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,4.5999999999999996],"tickmode":"array","ticktext":["07_Harvey-Samuel2023","05_Harvey-Samuel2023","06_Harvey-Samuel2023","04_Harvey-Samuel2023"],"tickvals":[1,2,3,4],"categoryorder":"array","categoryarray":["07_Harvey-Samuel2023","05_Harvey-Samuel2023","06_Harvey-Samuel2023","04_Harvey-Samuel2023"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Unique cross ID","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Nuclease transgene inheritance","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b462412c2c":{"x":{},"y":{},"size":{},"type":"scatter"},"52b42f99347b":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b462412c2c","visdat":{"52b462412c2c":["function (y) ","x"],"52b42f99347b":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: Cross Number_A unique short identifier of the source publication.</p>
<p>Y-axis: Nuclease transgene inheritance</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_carrying_progeny_from_doublehetd1.sex" class="slide level2 smaller">
<h2>Nuclease_Carrying_Progeny_From_DoubleHet:D1.Sex</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-04b65a1d3a4e6678ca5d" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-04b65a1d3a4e6678ca5d">{"x":{"data":[{"x":[0.94689438003115356,2.0720762838609517,null,0.97724423045292497,2.0957543510012329],"y":[0.49771689497716892,0.49433962264150944,null,0.50964630225080387,0.47300215982721383],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.94689438003115356,2.0720762838609517,0.97724423045292497,2.0957543510012329],"y":[0.49771689497716892,0.49433962264150944,0.50964630225080387,0.47300215982721383],"text":["Harvey-Samuel2023_06_D0.M.D1.F.kmo (49.8%:763/1533)<br>Total (49.8%:763/1533)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (49.4%:131/265)<br>Total (49.4%:131/265)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (51%:951/1866)<br>Total (51%:951/1866)","Harvey-Samuel2023_05_D0.F.D1.M.kmo (47.3%:219/463)<br>Total (47.3%:219/463)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[17.752922883359997,7.3695752943754629,19.587540214402548,9.7490316265098027],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Female","Male"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Female","Male"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Sex of the drive carrying parent","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Nuclease transgene inheritance","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b41c0c3f":{"x":{},"y":{},"size":{},"type":"scatter"},"52b43ad05b1f":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b41c0c3f","visdat":{"52b41c0c3f":["function (y) ","x"],"52b43ad05b1f":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: D1 is the F1 parent that is being evaluated, and D1.Sex is the sex of that group/individual. In almost all cases, one sex is of interest for the gene drive process in the cross (e.g., drive-carrying males crossed to wild-type females). In most crosses, this is an individual or group of individuals that are double heterozygous for the gRNA and Cas9 drive element. For preparation / control crosses (e.g., gRNA individuals crossed to Cas9 carriers), the sex carrying the element designed to be biased is designated as D1.</p>
<p>Y-axis: Nuclease transgene inheritance</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<section id="nuclease_carrying_progeny_from_doublehetd0.nuclease.sex" class="slide level2 smaller">
<h2>Nuclease_Carrying_Progeny_From_DoubleHet:D0.Nuclease.Sex</h2>
<div class="cell-output-display">
<div style="height: 575px" fig.align="center">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-a604482c5e6c0c839b38" style="width:100%;height:400px;"></div>
<script type="application/json" data-for="htmlwidget-a604482c5e6c0c839b38">{"x":{"data":[{"x":[0.94689438003115356,2.0720762838609517,null,0.97724423045292497,2.0957543510012329],"y":[0.47300215982721383,0.49433962264150944,null,0.50964630225080387,0.49771689497716892],"text":"","type":"scatter","mode":"lines","line":{"width":0.56692913385826771,"color":"rgba(0,0,0,0.8)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.94689438003115356,2.0720762838609517,0.97724423045292497,2.0957543510012329],"y":[0.47300215982721383,0.49433962264150944,0.50964630225080387,0.49771689497716892],"text":["Harvey-Samuel2023_05_D0.F.D1.M.kmo (47.3%:219/463)<br>Total (47.3%:219/463)","Harvey-Samuel2023_04_D0.M.D1.M.kmo (49.4%:131/265)<br>Total (49.4%:131/265)","Harvey-Samuel2023_07_D0.F.D1.F.kmo (51%:951/1866)<br>Total (51%:951/1866)","Harvey-Samuel2023_06_D0.M.D1.F.kmo (49.8%:763/1533)<br>Total (49.8%:763/1533)"],"customdata":["https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656","https://doi.org/10.1101/2023.06.12.544656"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,158,115,1)","opacity":0.59999999999999998,"size":[9.7490316265098027,7.3695752943754629,19.587540214402548,17.752922883359997],"symbol":"circle","line":{"width":0.94488188976377963,"color":"rgba(0,0,0,1)"}},"hoveron":"points","name":"(C. quinquefasciatus,1)","legendgroup":"(C. quinquefasciatus,1)","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":28.988330796549977,"r":7.3059360730593621,"b":57.288870439555382,"l":69.672063096720663},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"white","font":{"color":"rgba(0,0,0,1)","family":"","size":22.581984225819845},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.40000000000000002,2.6000000000000001],"tickmode":"array","ticktext":["Female","Male"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["Female","Male"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":25,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":true,"gridcolor":"rgba(190,190,190,1)","gridwidth":0.0033208800332088012,"zeroline":false,"anchor":"y","title":"Sex of grandparent carrying the nuclease gene","hoverformat":".2f","titlefont":{"size":20}},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.16000000000000003,1.1600000000000001],"tickmode":"array","ticktext":["0","0.25","0.5","0.75","1"],"tickvals":[0,0.25,0.5,0.74999999999999989,1],"categoryorder":"array","categoryarray":["0","0.25","0.5","0.75","1"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.6529680365296811,"tickwidth":0.66417600664176002,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176002,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Nuclease transgene inheritance","hoverformat":".2f","titlefont":{"size":20}},"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.8897637795275593,"font":{"color":"rgba(0,0,0,1)","family":"","size":18.065587380655877}},"hovermode":"closest","barmode":"relative","autosize":true,"width":1040,"height":550},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"52b49b8ddd":{"x":{},"y":{},"size":{},"type":"scatter"},"52b43d843ffd":{"x":{},"y":{},"size":{},"fill":{},"text":{},"customdata":{}}},"cur_data":"52b49b8ddd","visdat":{"52b49b8ddd":["function (y) ","x"],"52b43d843ffd":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>X-axis: The sex of the nuclease-carrying grandparent.</p>
<p>Y-axis: Nuclease transgene inheritance</p>
<p>O-O: Lines are drawn between paired crosses that are identical for the other essential factors, apart from the respective X-axis factor.</p>
</section>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<div class="smaller">

</div>
<section id="final-notes" class="slide level2">
<h2>Final notes</h2>
<ul>
<li><p>The systematic and detailed diagrams were useful for understanding the crosses that were performed. But probably too detailed for the general nature communications audience.</p></li>
<li><p>Homing the rescue white into a recipient chromosome carrying a separate white KO is very creative and resourceful. However, they could also, in males, have used sex of the progeny as a marker of homing.</p></li>
<li><p>My expectation is that the meta-analysis will still give a very broad confidence interval for the performance of gene drives in Cx. q. Only a single Cas9 line was tested, and there are also a few examples of poorly performing Cas9 lines in Anopheles.</p></li>
<li><p>Looking at molecular hallmarks of homing is more sensitive than just measuring overall inheritance bias. But does the distinction between 0% bias or ‘almost’ 0% matter that much?</p></li>
<li><p>A few typos in main figures.</p></li>
<li><p>Answering why it didn’t work by looking at cut-rate and timing of nuclease activity would potentially have provided more of a foundation for follow-up work.</p></li>
<li><p>The paper I will point to if someone wants to an example of how to best statistically analyse gene drive data.</p></li>
<li><p>Treating the egg rafts from male drive crosses as individual crosses is a bit cheeky. A single male may have mated multiple females.</p></li>
</ul>
</section>
<section id="homing-gene-drive-inheritance-rates-for-similar-transgenes-have-varied-between-species.-1" class="slide level2">
<h2>Homing gene drive inheritance rates for similar transgenes have varied between species.</h2>

<img data-src="Harvey-Samuel2023/Z01%20Homing%20gene%20drive%20inheritance%20rates%20for%20similar%20transgenes%20have%20varied%20between%20species..PNG" class="r-stretch quarto-figure-center"><div class="footer footer-default">

</div>
</section>
    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="index_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="index_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="index_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="index_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="index_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="index_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="index_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="index_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="index_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="index_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'smaller': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.togglePdfExport(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: false,

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'none',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1050,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 2.0e-2,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const isCodeAnnotation = (el) => {
        for (const clz of el.classList) {
          if (clz.startsWith('code-annotation-')) {                     
            return true;
          }
        }
        return false;
      }
      const clipboard = new window.ClipboardJS('.code-copy-button', {
        text: function(trigger) {
          const codeEl = trigger.previousElementSibling.cloneNode(true);
          for (const childEl of codeEl.children) {
            if (isCodeAnnotation(childEl)) {
              childEl.remove();
            }
          }
          return codeEl.innerText;
        }
      });
      clipboard.on('success', function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      });
      function tippyHover(el, contentFn) {
        const config = {
          allowHTML: true,
          content: contentFn,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start'
        };
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          return note.innerHTML;
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>